test/SFSmartSyncTestSuite.js

```diff
@@ -2756,7 +2756,7 @@ SmartSyncTestSuite.prototype.testSyncUpLocallyUpdated = function() {
     var self = this;
     var idToName = {};
     var updatedRecords;
-    var options = {fieldlist: ["Name"]};
+    var options = {fieldlist: ["Name"], mergeMode: Force.MERGE_MODE_DOWNLOAD.OVERWRITE};
```

Comment:
Maybe the enum Force.MERGE_MODE_DOWNLOAD needs to be renamed since we are using it also for upload?